sites/avivator/src/state.js

```diff
@@ -152,10 +152,9 @@ export const useViewerStore = create(set => ({
 }));
 
 export const useLoader = () => {
-  const [fullLoader, image] = useChannelsStore(store => [
-    store.loader,
-    store.image
-  ]);
+  const [fullLoader, image] = useChannelsStore(
+    useShallow(store => [store.loader, store.image])
```

Comment:
No familiar with this library, but this change looks different from the others. Why is the shallow needed here but not before?

Reply:
> or where applicable (returning new references)

I got the error which the upgrade mentioned without it, and this is a new reference